modern/org/commcare/modern/database/TableBuilder.java

```diff
@@ -180,7 +180,7 @@ public String getTableCreateString() {
 
     //sqlite doesn't like dashes
     public static String scrubName(String input) {
-        return input.replace("-", "_");
+        return input.replace("-", "_dash_").replace(".", "_dot_").replace("+", "_plus_");
```

Comment:
Yeah, and it would only crash if you try to update CommCare with an app installed from `2.30` to the dev build. Otherwise the tables would just get created with the new naming structure.